docs/images/images.asciidoc

```diff
@@ -0,0 +1,7 @@
+tag::java-env-vars[]
+image:./java-lambda-env-vars.png[Java environment variables configuration section in AWS Console]
+end::java-env-vars[]
```

Comment:
FYI @AlexanderWert -- we can't include images across repos, but we can include tagged regions. Adding this file allows us to include the tagged region in the agent docs instead of the image.